omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -210,12 +264,102 @@ Copy the `systemd.service` file, then enable and start the service::
 
 
 
-Maintenance
------------
+Maintening OMERO.web
+--------------------
+
+**The following steps are run as the omero-web system user.**
+
+If an attempt is made to access OMERO.web whilst it is not running, the generated NGINX configuration file will automatically display a maintenance page.
+
+    -  Session cookies :property:`omero.web.session_expire_at_browser_close`:
```

Comment:
This begins by talking about `omero.web.session_expire_at_browser_close`, but ends up also talking about `omero.web.session_cookie_age`. Is there a particular reason to highlight these two properties?

Reply:
This was in the layout of the original page.